package cms_8120;
import "type/cmd.proto";
option java_package = "com.hikvision.finance.fms.cms.hpp.center";
option java_outer_classname = "SysconfigProto";
option java_generic_services = true;



//系统参数表
message Sysconfig
{
    //必须字段
    optional int32 id                       = 1;    // 在数据库中的流水号
    optional int32 nKey                     = 2;    // 参数key值
    optional string strValue                = 3;    // 参数值
    optional string strComment              = 4;    // 内容
    optional int32 nEnabled                 = 5;    // 是否可用
    optional int32 nType                    = 6;    // 类型
    optional string strDescribe             = 7;    // 说明
    optional string strUpdatetime           = 8;    // 更新时间
}

message SysconfigList
{
    optional int32 cmd                      = 1;    // CMD_DT_SYSCONFIG_LIST
    repeated Sysconfig sysconfig            = 2;    // 中心系统参数
}


service ModifySysconfigService {
	
	rpc QyerySysconfigServicetime(Sysconfig) returns (SysconfigList);			// CMD_DT_SYSCONFIG_LIST--获取录像守卫服务的工作时间
	
}